doc/overview.rst

```diff
@@ -264,6 +264,8 @@ When run by ``PopenSpawn``, they may behave differently.
 
 .. seealso::
 
-   `winpexpect <https://pypi.python.org/pypi/winpexpect>`__ and `wexpect <https://gist.github.com/anthonyeden/8488763>`__
-     Two unmaintained pexpect-like modules for Windows, which work with a
-     hidden console.
+   `wexpect <https://pypi.org/project/wexpect/>`__ is a maintained
+   alternative for Windows. It supports pyton3.x
```

Comment:
Glad to hear it, but I don't think we need to explicitly mention Python 3 support in 2020 - I'd assume that any maintained module supports Python 3 unless it says otherwise.

Reply:
However, this is the only Windows alternative, with Python 3 support, I let it go, but the link is essential. The old link of wexpect points to an unmaintained repo, while the new link shows the maintained package. (The maintained package's source is https://github.com/raczben/wexpect)

Reply:
Yep, I have no objection to updating the link.